modules/nodes-pods-configuring-pod-disruption.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-configuring.adoc
+
+[id='nodes-pods-configuring-pod-distruption_{context}']
+= Specifying the number of pods that must be up
+
+A _pod disruption budget_ is part of the
```

Comment:
This looks more like a module about pod description budgets than a procedure intro.

Reply:
@kalexand-rh Thanks. I think i learned as I went along that the procedure topic text needs to brief.